templates/404.html

```diff
@@ -9,9 +9,15 @@
             <img src="/static/images/400art.svg" alt=""/>
             <div class="errorbox">
                 <div class="errorcontent">
+                    {% if status_code == 405 %}
+                    <h1 class="lead">Method not allowed (405)</h1>
```

Comment:
This message duplication / if else block is unnecessary, you can just embed the status code with `{{status_code}}`.

Reply:
I am passing context with `status_code=405` which is not used while rendering `404.html`, so we can use shorthand conditional for email part (like this `{{ '405' if status_code == 405 else '404' }}` ) and leave the `lead` `h1` (with conditional if statement) as it is.
What do you think?

Reply:
I see. We should keep the different leads, so I'd move the if else block into the `<h1>` tag and use the inline conditional in the mailto subject. This way we get rid of the message duplication.

Reply:
Apparently, in `custom_checks.py` `html_rules` have a pattern `'={'`, when found it gives linting error with description **Likely missing quoting in HTML attribute**.
Now it finds the same pattern when I added inline conditional like: 
`?Subject={{ '405' if status_code == 405 else '404 }}` and give the same error.

Reply:
We can always exclude a linter error via a line in `tools/linter_lib/custom_check.py`; it's not intended to cover this situation.